python/lsst/ts/ess/ess_csc.py

```diff
@@ -347,39 +384,59 @@ def add_arguments(cls, parser: argparse.ArgumentParser) -> None:
         )
         super().add_arguments(parser)
 
-    def _get_device(self) -> None:
+    def _get_device(self, device_configuration: DeviceConfig) -> Optional[Any]:
         """Get the device to connect to by using the configuration of the CSC
         and by detecting whether the code is running on an aarch64 architecture
         or not.
 
+        Parameters
+        ----------
+        device_configuration: `dict`
+            A dict representing the device to connect to. The format of the
+            dict follows the configuration of the ts_ess project.
```

Comment:
Do you mean the configuration for the ESS CSC? If so, please be specific, e.g. `lsst.ts.ess.CONFIG_SCHEMA` if that is the relevant constant.

Reply:
Done.